src/main/java/fivium/pat/utils/PatUtils.java

```diff
@@ -50,6 +58,7 @@ public static String getUserIdFromJWT(String token) {
 		String subject = claimsBody.getSubject();
 		return subject;
 	}
+	
```

Comment:
new line?

Reply:
Will tidy up in future commit

Reply:
Side effect of moving across code from RnsUtils